packages/procaptcha-bundle/src/index.tsx

```diff
@@ -20,12 +20,16 @@ import {
     ProcaptchaOutput,
 } from '@prosopo/types'
 import { Procaptcha } from '@prosopo/procaptcha-react'
+import { ProcaptchaPow } from '@prosopo/procaptcha-pow'
 import { at } from '@prosopo/util'
 import { createRoot } from 'react-dom/client'
 
+const features = ['image', 'pow'] as const
```

Comment:
this needs to be moved to elsewhere outside of procaptcha-bundle

Reply:
Fixed in latest push